scripts/build_android.sh

```
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ex;
+
+export VERSION=$(grep '^version:' pubspec.yaml | sed -e 's/version: //')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)